app/cells/decidim/challenges/challenge_list_item_cell.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Challenges
+    # This cell renders a horizontal challenge card
+    # for an given instance of a challenge in a challenges list
+    class ChallengeListItemCell < Decidim::ViewModel
+      include ActiveSupport::NumberHelper
+      include Decidim::LayoutHelper
+      include Decidim::ActionAuthorizationHelper
+      include Decidim::Challenges::ApplicationHelper
+      include Decidim::Challenges::Engine.routes.url_helpers
+
+      delegate :current_user, :current_settings, :current_order, :current_component, :current_participatory_space, to: :parent_controller
+
+      private
+
+      def challenge_path
+        resource_locator(model).path
+      end
+
+      def show
+        render
+      end
+
+      def resource_path
+        resource_locator(model).path
+      end
+
+      def resource_title
+        translated_attribute model.title
+      end
+
+      def resource_description
+        translated_attribute model.global_description
+      end
+
+      def resource_sdg
+        sgds = [
```

Comment:
Maybe this could be changed for something like:
```ruby
    Decidim::Sdgs::Sdg::SDGS.map do |sdg_name|
      [I18n.t("#{sdg_name}.objectives.subtitle", scope: "decidim.components.sdgs")]
    end
```
this way, if sdgs are changed some day, this method always will return current ones.